### PR TITLE
Add description to tag

### LINKED ISSUE #21: A user should be able to see a tag with a description
The idea of this requirement is that we should add a description for tags so that someone that comes in can know what the tags are used for.

### PR DESCRIPTION
**Pull request by brentm5**
Closes #21

### CODE REVIEW

File: app/controllers/tags_controller.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def new
   end
 
   def create
-    @tag = Tag.new(params.require(:tag).permit(:name))
+    @tag = Tag.new(params.require(:tag).permit(:name, :description))
```
Comment (drapergeek) [inline app/controllers/tags_controller.rb]: I would extract the params at this point.  No point in this line doing this much.

File: db/migrate/20130322031239_add_description_to_tag.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddDescriptionToTag < ActiveRecord::Migration
+  def change
+    add_column :tags, :description, :text, default: ''
```
Comment (drapergeek) [inline db/migrate/20130322031239_add_description_to_tag.rb:3]: I like defaults :P

**Comment by drapergeek** [comment]
:+1:

File: db/migrate/20130322031239_add_description_to_tag.rb (hunk 2 above)
Comment (brentm5) [inline db/migrate/20130322031239_add_description_to_tag.rb:3]: Is that sarcasm?

Comment (drapergeek) [inline db/migrate/20130322031239_add_description_to_tag.rb:3]: No not at all. If it needs a default, set a default.

**Pull request closed without merging**